packages/ember-engines/tests/dummy/config/ember-try.js

```diff
@@ -47,6 +45,24 @@ module.exports = async function () {
         npm: {
           devDependencies: {
             'ember-source': '~5.4.0',
+            '@ember/string': '*',
+            'ember-engines-router-service': '*',
```

Comment:
@void-mAlex this is strange it's added only here and not added to other scenarios.

Also I thought you would want `ember-engines-router-service` in `devDependencies` of `ember-engines` package - otherwise how does it get pulled into build at all?